source/jobs/pause-resume-kafka-jobs.txt

```diff
@@ -0,0 +1,75 @@
+.. _rm-pause-resume-kafka-jobs:
+
+================================
+Pause and Resume Kafka Sync Jobs
+================================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+When you have an Apache Kafka supported sync job, you can pause and 
+resume the sync job. You may want to pause a sync job for maintenance 
+windows or planned outages on your source database. 
+
+About this Task
+---------------
+
+- You can only pause and resume **Kafka** or **Confluent Cloud Kafka** 
```

Comment:
[Suggestion]

[Per the style guide](https://www.mongodb.com/docs/meta/style-guide/style/text-formatting) we try to not to bold the Product/feature names but maybe to still keep the emphasis we can do the following: 

```suggestion
- You can **only** pause and resume Kafka or Confluent Cloud Kafka
```